src/main/java/seedu/address/MainApp.java

```diff
@@ -57,8 +49,10 @@ public void init() throws Exception {
 
         UserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(config.getUserPrefsFilePath());
         UserPrefs userPrefs = initPrefs(userPrefsStorage);
-        AddressBookStorage addressBookStorage = new JsonAddressBookStorage(userPrefs.getAddressBookFilePath());
-        storage = new StorageManager(addressBookStorage, userPrefsStorage);
+        seedu.address.storage.VolunteerStorage volunteerStorage = new JsonVolunteerStorage(userPrefs.getVolunteerStorageFilePath());
+        seedu.address.storage.EventStorage eventStorage = new JsonEventStorage(userPrefs.getEventStorageFilePath());
```

Comment:
change to `VolunteerStorage` and `EventStorage` respectively since you have already imported both in `MainApp.java`